itest/opstackl2/op_test_manager.go

```diff
@@ -52,16 +56,21 @@ type BaseTestManager = base_test_manager.BaseTestManager
 
 type OpL2ConsumerTestManager struct {
 	BaseTestManager
-	BabylonHandler    *e2eutils.BabylonNodeHandler
-	BabylonFpApp      *service.FinalityProviderApp
-	ConsumerFpApps    []*service.FinalityProviderApp
-	EOTSServerHandler *e2eutils.EOTSServerHandler
-	BaseDir           string
-	SdkClient         *sdkclient.SdkClient
-	OpSystem          *ope2e.System
+	BabylonHandler       *e2eutils.BabylonNodeHandler
+	BabylonFpApp         *service.FinalityProviderApp
+	ConsumerFpApps       []*service.FinalityProviderApp
+	EOTSServerHandler    *e2eutils.EOTSServerHandler
+	BaseDir              string
+	FinalityGadget       *finalitygadget.FinalityGadget
+	FinalityGadgetClient *fgclient.FinalityGadgetGrpcClient
+	Db                   db.IDatabaseHandler
+	DbPath               string
+	OpSystem             *ope2e.System
 }
 
 func StartOpL2ConsumerManager(t *testing.T, numOfConsumerFPs uint8) *OpL2ConsumerTestManager {
+	defaultlog.SetOutput(os.Stdout)
```

Comment:
just curious, why do we need this?

Reply:
will delete - was playing around with different logging settings previously and forgot to revert